phoenix-tools/src/main/java/org/apache/phoenix/schema/SchemaExtractionProcessor.java

```diff
@@ -202,12 +202,20 @@ protected String extractCreateViewDDL(PTable table) throws SQLException {
 
     private String generateCreateViewDDL(String columnInfoString, String baseTableFullName,
             String whereClause, String pSchemaName, String pTableName) {
-        String viewFullName = SchemaUtil.getQualifiedTableName(pSchemaName, pTableName);
+        String viewFullName = getPTableFullName(pSchemaName, pTableName);
         StringBuilder outputBuilder = new StringBuilder(String.format(CREATE_VIEW, viewFullName,
                 columnInfoString, baseTableFullName, whereClause));
         return outputBuilder.toString();
     }
 
+    private String getPTableFullName(String pSchemaName, String pTableName) {
```

Comment:
Why not add this function to SchemaUtil class itself?

Reply:
This is not needed in for internals for phoenix.

Reply:
This is not needed in internals of phoenix.

Reply:
Maybe so today, but we never know when it might be needed later. SchemaUtil seems like the right place for it instead of fragmented/duplicated in various tools.